fifa_draft/utils.py

```diff
@@ -42,7 +52,7 @@ def team_form_validation(request, form, profile):
         messages.error(request, "Featured image is too big (max 3mb)")
```

Comment:
W tym miejscu, jeżeli kod wejdzie do tego `else` to nie ma żadnego returna. A type hint mówi, że funkcja ZAWSZE zwraca boola. Więc albo jest błąd w kodzie, albo w type hincie.

Jeżeli funkcja może zwracać boola, albo nic, to wtedy type hint to `Optional[bool]` (`from typing import Optional`).